src/main/kotlin/com/ampnet/identityservice/service/impl/UserServiceImpl.kt

```diff
@@ -150,4 +156,12 @@ class UserServiceImpl(
             }
         }
     }
+
+    private fun isDocumentExpired(user: User): Boolean =
```

Comment:
You don't need the entire `user` object here. Also, the place where you use this already checks for `user.userInfoUuid != null`, so if you pass in `userInfoUuid` to this function you can already be sure that it isn't `null`.